### PR TITLE
Doin't use type hint for $user param of discord_user_groups_changed

### PR DESCRIPTION
**Pull request by paladox**
This can return User or UserRightsProxy (thus you will get "must be an instance of User, instance of UserRightsProxy given" if it returns UserRightsProxy)

Echo doesn't use type hints either for this.

### CODE REVIEW

File: DiscordNotificationsCore.php [hunk 1]
```diff
@@ -348,7 +348,7 @@ static function discord_user_groups_changed(User $user, array $added, array $rem
 		$message = sprintf(
             self::getMessage('discordnotifications-change-user-groups'),
 			self::getDiscordUserText($performer),
-			self::getDiscordUserText($user),
+			self::getDiscordUserText($user->getName()),
```
Comment (paladox) [inline DiscordNotificationsCore.php]: This is to support UserRightsProxy (this is compatible with User which also returns this too).

**Comment by paladox** [comment]
Bump

**Comment by kulttuuri** [comment]
Tested to be working and merging now. Thank you! :)

**Comment by kulttuuri** [comment]
Could you verify that the commits went in correctly to master?

**Comment by paladox** [comment]
Looks correct to me, thanks!

**Comment by kulttuuri** [comment]
Fantastic. Thank you!

**Pull request merged**